### ISSUE TITLE
Accessibility][On Focus - Bot Responses-More Option]: Tab focus does not visible anywhere after clicking enter on ‘Edit’ menu item.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2051]
**User Experience:**
Keyboard dependent users would face difficulty if focus does not visible anywhere on the page/dialog that opens up after selecting any control.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Select 'Bot Responses' link from left navigation.
3. Navigate within the table using tab key.
4. Press right arrow key & navigate up to 'More option' button & activate it.
5. Select 'Edit' menu item and observe the issue.

**Actual Result:**
Tab focus does not visible anywhere after clicking enter on ‘Edit’ menu item. User has to press tab key, then focus comes within the edit field.

**Note:**
Same issue is repro on clicking 'Main' button that comes within 'Used in' column.

**Expected Result:**
Tab focus should automatically jump within the edit field after clicking enter on 'Edit’ menu item.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/CELA/Official_Microsoft_Accessibility_Standards/MAS%203.2.1%20%E2%80%93%20On%20Focus.docx?d=wa3e028a828194cdab901ec19ff068aa6&csf=1&e=mVFpic

**Attachment:**
[MAS3.2.1_More option_Edit.zip](https://github.com/microsoft/BotFramework-Composer/files/4239185/MAS3.2.1_More.option_Edit.zip)

**Comment by cwhitten** [comment]
addressed via https://github.com/microsoft/BotFramework-Composer/pull/2265

**Comment by cwhitten** [comment]
Mistakenly closed.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
